src/Console/NotifySchedule.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of fof/best-answer.
+ *
+ * Copyright (c) 2019 FriendsOfFlarum.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FoF\BestAnswer\Console;
+
+use Flarum\Foundation\Paths;
+use Flarum\Settings\SettingsRepositoryInterface;
+use Illuminate\Console\Scheduling\Schedule;
+
+class NotifySchedule
+{
+    public function __invoke()
+    {
+        $schedule = app()->make(Schedule::class);
```

Comment:
If possible, these should be injected in the constructor.

Reply:
Will do!